cfy_manager/components/restservice/db.py

```diff
@@ -147,6 +176,21 @@ def create_amqp_resources(configs=None):
     logger.notice('AMQP resources successfully created')
 
 
+@_connect_to_db
+def check_manager_in_table(connection):
+    try:
+        result = \
```

Comment:
we tend to wrap in '()' to avoid the '\'

Reply:
Fixed